### DISCUSSION TITLE
Template override

### DISCUSSION THREAD

**Discussion by eduardoalonsoalbella** [craftcms/cms#9926] [Ideas] [answered]
Hi to everyone, I would like to know if it would be possible to modify the method _resolveTemplateInternal of craft\web\View to priorityze the roots path over the basepath(templatesPath), it would allow to be able to change specific templates completely, and only if not found on another location it would try to load the default template.

I would like to replace the template src/templates/_elements/sitemenu.html with a custom one, but it seems it's not possible to change an existing template in the base folder with a custom one.

Thanks

**Comment by brandonkelly** [comment, marked as answer]
This is not something we would like to support. If there is a specific thing you’d like to see changed about the `sitemenu` template, please request that in a separate issue and we can consider it.

**Reply by eduardoalonsoalbella** [reply]
The downside of this approach is that modules/plugins could be trying to replace the same template

**Comment by eduardoalonsoalbella** [comment]
Thanks, done

https://github.com/craftcms/cms/discussions/9932
